client.go

```diff
@@ -266,6 +270,9 @@ func (c *Client) Close() error {
 //
 // If no ProcessAt or ProcessIn options are provided, the task will be pending immediately.
 func (c *Client) Enqueue(task *Task, opts ...Option) (*TaskInfo, error) {
+	if strings.TrimSpace(task.Type()) == "" {
+		return nil, fmt.Errorf("%w", ErrEmptyTypename)
```

Comment:
this should be sufficient, since we don't want to export a sentinel error value for this case.
```go
return nil, fmt.Errorf("task typename cannot be empty")
```

Reply:
Ok, now I see what you meant. Thanks for clarifying.